tag/nitag.yml

```diff
@@ -0,0 +1,1814 @@
+      operationId: GetTagMaxAggregate
+      responses:
+        200:
+          description: Maximum aggregate value
+          schema:
+            type: string
+        204:
+          description: No aggregate value found
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          description: Tag with the given path was not found
+        default:
+          $ref: '#/responses/Error'
+      parameters:
+        - in: path
+          name: path
+          description: Tag path
+          type: string
+          required: true
+  /v2/tags/{path}/values/aggregates/mean:
+    get:
+      tags:
+        - tagAggregates
+      summary: Get the value of a tag's "avg" aggregate
+      description: Get the value of a tag's "avg" aggregate
+      operationId: GetTagMeanAggregate
+      responses:
+        200:
+          description: Mean aggregate value
+          schema:
+            type: string
+        204:
+          description: No aggregate value found
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          description: Tag with the given path was not found
+        default:
+          $ref: '#/responses/Error'
+      parameters:
+        - in: path
+          name: path
+          description: Tag path
+          type: string
+          required: true
+  /v2/tags/{path}/values/aggregates/count:
+    get:
+      tags:
+        - tagAggregates
+      summary: Get the value of a tag's "count" aggregate
+      description: Get the value of a tag's "count" aggregate
+      operationId: GetTagCountAggregate
+      parameters:
+        - in: path
+          name: path
+          description: Tag path
+          type: string
+          required: true
+      responses:
+        200:
+          description: Count aggregate value
+          schema:
+            type: string
+        204:
+          description: No aggregate value found
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          description: Tag with the given path was not found
+        default:
+          $ref: '#/responses/Error'
+  /v2/tags/{path}/reset-aggregates:
+    post:
+      tags:
+        - tagAggregates
+      summary: 'Reset the aggregates of a tag'
+      description: Reset tag aggregates
+      operationId: ResetTagAggregates
+      parameters:
+        - in: path
+          name: path
+          description: Tag path
+          type: string
+          required: true
+      responses:
+        200:
+          description: Success
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          description: Tag with the given path was not found
+        default:
+          $ref: '#/responses/Error'
+  /v1/tags-rpc/update-current-values:
+    post:
+      tags:
+        - tags
+      summary: Update the current values of multiple tags
+      description: Push multiple current value updates to multiple tags in one request. Note that this route is deprecated.
```

Comment:
Updates multiple tags to their current value. This route is deprecated. [Use x instead?]

Same question as before about how we should handle deprecated routes.